### ISSUE TITLE
Provide an Arch repository for your packages

### ISSUE THREAD

**Issue by GamePlayer-8** [cwt-vf2/archlinux-image-vf2#16]
Good afternoon,

Is there a possibility for providing an Arch repository for your RTC, GPU, kernel packages etc., or the [AUR](https://aur.archlinux.org)?

**Comment by cwt** [comment]
Unfortunately, I cannot publish these packages to the AUR:

The AUR only accepts packages for x86_64 architecture or architecture-independent (any) packages, as per the [Arch package guidelines](https://wiki.archlinux.org/title/Arch_package_guidelines#Architectures). Since my packages are built for riscv64, they are not eligible for inclusion. I attempted to publish one previously, but it was removed for this reason.

For setting up an alternative repository would require maintaining a 24/7 server or VM with sufficient storage capacity. Without sponsorship, this would incur additional expenses that I cannot currently afford.
